camayoc/qcs_models.py

```diff
@@ -305,10 +342,11 @@ def equivalent(self, other):
                 # the list of id's we used to create the source
                 for cred in other_creds:
                     cred_ids.append(cred.get('id'))
-                if sorted(value) != sorted(cred_ids):
+                if sorted(local_items.get(key)) != sorted(cred_ids):
```

Comment:
This may raise:

```
In [4]: sorted(None)
------------------------------------------------------------
TypeError                  Traceback (most recent call last)
<ipython-input-4-f48f477b5e8b> in <module>()
----> 1 sorted(None)

TypeError: 'NoneType' object is not iterable
```

Maybe `sorted(local_items[key])` will produce a more meaningful error (KeyError) when the key we are looking for is missing.

Reply:
good call